rfcs/meta_yml_notify.md

```diff
@@ -0,0 +1,50 @@
+# RFC 37: Split META.yml `suggested_reviewers` into `suggested_reviewers` and `notify`
+
+## Summary
+
+Change `META.yml` files in wpt to have `notify` in addition to `suggested_reviewers`.
+@wpt-pr-bot mentions everyone in `notify` and then requests review from one person from `suggested_reviewers`.
+
+## Details
+
+At TPAC 2019 we set out some [priorities for 2020](https://docs.google.com/document/d/1gie7LFb6cAUfabY86MYuWM7m7ux_FaKhDkLdpz0zWkg/edit),
+which include increasing PR review velocity and reduce the number of stale PRs.
+
+wpt has META.yml files with `suggested_reviewers`. @wpt-pr-bot will request review from everyone listed in `suggested_reviewers` for PRs that change that directory, and also assign one of them in a round-robin fashion.
```

Comment:
Is implementing the round robin review part of this RFC?

Reply:
Yes, clarified below.